homeassistant/config.py

```diff
@@ -263,12 +268,17 @@ def process_ha_config_upgrade(hass):
 
 
 @asyncio.coroutine
-def async_process_ha_core_config(hass, config):
+def async_process_ha_core_config(hass, config, complete_config=None):
     """Process the [homeassistant] section from the config.
 
     This method is a coroutine.
     """
     config = CORE_CONFIG_SCHEMA(config)
+
+    if complete_config:
```

Comment:
This should not be part of `process_ha_core_config`, it seems like something that should live in a standalone package.